src/Plugin/WebformElement/CivicrmOptions.php

```diff
@@ -215,4 +215,21 @@ public static function ajaxCallback(array $form, FormStateInterface $form_state)
     return $element;
   }
 
+  /**
+   * {@inheritdoc}
+   */
+  public function hasMultipleWrapper() {
+    return FALSE;
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function hasMultipleValues(array $element) {
+    if (!empty($element['#extra']['multiple']) || (!empty($element['#options']) && count($element['#options']) === 1)) {
+      return TRUE;
+    }
+    return FALSE;
+  }
+
 }
```

Comment:
Going to lift these out of this PR - and solve the Notice separately: 

`Array to string conversion in Drupal\webform\WebformSubmissionStorage->saveData() (line 1343 of /Applications/MAMP/htdocs/d9civicrm.local/web/modules/contrib/webform/src/WebformSubmissionStorage.php)`